models/git/branch.go

```diff
@@ -101,9 +101,9 @@ func (err ErrBranchesEqual) Unwrap() error {
 type Branch struct {
 	ID            int64
 	RepoID        int64  `xorm:"UNIQUE(s)"`
-	Name          string `xorm:"UNIQUE(s) NOT NULL"`
+	Name          string `xorm:"UNIQUE(s) NOT NULL"` // git's ref-name is case-sensitive internally, however, in database, it's case-insensitive at the moment
 	CommitID      string
-	CommitMessage string `xorm:"TEXT"`
+	CommitMessage string `xorm:"TEXT"` // it only stores the message summary (the first line)
```

Comment:
:fearful:
Those two comments don't sound like a good idea…

Reply:
I don't like them, but that's the fact.

 Sync branches into databases #22743 
 Fix branch commit message too long problem #25588